source/modules/MEI.xml

```diff
@@ -1569,13 +1569,7 @@
       </valList>
     </content>
   </macroSpec>
-  <macroSpec ident="data.IDREF" module="MEI" type="dt">
```

Comment:
Reason for removal?

Reply:
It would be good to see the commit messages – it's not used anywhere…